tools/docker/Dockerfile.jinja

```diff
@@ -54,6 +54,7 @@ RUN git clone https://github.com/asterinas/lmbench.git
 RUN wget https://www.iozone.org/src/current/iozone3_506.tar
 RUN tar -x -f iozone3_506.tar
 RUN git clone -b fio-3.37 https://github.com/axboe/fio.git
+RUN git clone https://kernel.googlesource.com/pub/scm/linux/kernel/git/mason/schbench
```

Comment:
Now that the Dockerfile is updated, we need to build the new Docker image. This can be done by bumping the version of Asterinas  to 0.10.1 with `tools/bump_version.sh`. The bumped version will trigger Docker image build CI.